byte  ONLY_JOINTS_MODE = 0
byte  ONLY_BASE_MODE  = 1
byte JOINTS_AND_BASE_MODE = 2
byte ORIGINAL_MODE  = 3
